src/components/Editor/CallSites.js

```diff
@@ -142,17 +142,34 @@ class CallSites extends Component {
       });
     }
   }
+  filterCallSites() {
```

Comment:
I'd love a more specific name on this function; i.e. filtered by what?  Additionally, a comment above the function as to why the given filter is there would be nice.